kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQueryHelper.java

```diff
@@ -29,43 +29,64 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
 
 /**
  * Convenience class for creating a default {@link com.google.cloud.bigquery.BigQuery} instance,
  * with or without login credentials.
  */
 public class BigQueryHelper {
   private static final Logger logger = LoggerFactory.getLogger(BigQueryHelper.class);
+  private static String keyFileType;
 
   /**
    * Returns a default {@link BigQuery} instance for the specified project with credentials provided
    * in the specified file, which can then be used for creating, updating, and inserting into tables
    * from specific datasets.
    *
    * @param projectName The name of the BigQuery project to work with
-   * @param keyFilename The name of a file containing a JSON key that can be used to provide
+   * @param keyFile The name of a file containing a JSON key that can be used to provide
    *                    credentials to BigQuery, or null if no authentication should be performed.
    * @return The resulting BigQuery object.
    */
-  public BigQuery connect(String projectName, String keyFilename) {
-    if (keyFilename == null) {
+  public BigQuery connect(String projectName, String keyFile) {
```

Comment:
Since this isn't necessarily a file, it might be worth it to change the name of the `keyFile` parameter here. `keyString` or `key` or something else might be better.